### PR TITLE
add sticky bar

### PR DESCRIPTION
**Pull request by DmytroHryshyn**
adds sticky bar on the top level 
image

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -218,9 +218,17 @@ export async function activate(context: vscode.ExtensionContext) {
 					if (!viewProps) {
 						return;
 					}
+
+					const job = jobManager.getJob(jobHash);
+
+					if (!job) {
+						throw new Error('Unable to get job');
```
Comment (grzpab) [inline src/extension.ts:225]: `'Unable to find a job with the job hash ${jobHash}`

File: src/extension.ts [hunk 2]
```diff
@@ -309,9 +318,17 @@ export async function activate(context: vscode.ExtensionContext) {
 					if (!viewProps) {
 						return;
 					}
+
+					const job = jobManager.getJob(jobHashes[0]);
```
Comment (grzpab) [inline src/extension.ts:322]: is `jobHashes[0]` possibly undefined?

File: intuita-webview/src/jobDiffView/DiffViewer/Header/index.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import { VSCodeButton } from '@vscode/webview-ui-toolkit/react';
+
+import { ReactComponent as UnifiedIcon } from '../../../assets/Unified.svg';
+import { ReactComponent as SplitIcon } from '../../../assets/Split.svg';
+import { DiffViewType } from '../../../shared/types';
+
+import styles from './style.module.css';
+
+type Props = {
```
Comment (grzpab) [inline intuita-webview/src/jobDiffView/DiffViewer/Header/index.tsx:9]: `Readonly<`

**Comment by grzpab** [comment]
@DmytroHryshyn can you write in the header something like: `Executed the codemod: ${name or path}`
Also, the button on the right could use the phrase Inline / Side by Side inlined before the image.

**Comment by DmytroHryshyn** [comment]
> @DmytroHryshyn can you write in the header something like: `Executed the codemod: ${name or path}` Also, the button on the right could use the phrase Inline / Side by Side inlined before the image.

I've removed button titles intentionally, wanted to follow vscode design code (they just use icon+hover message)
image

**Review by grzpab — APPROVED** [review]

**Pull request merged**